pkg/version/version.go

```diff
@@ -9,7 +9,7 @@ const (
 )
 
 var (
-	knowVersions       = []string{}
+	knowVersions       []string
 	knowVersionIndexes = map[string]int{}
```

Comment:
This make `knownVersions` be `nil`, is that ok?
Can be `knownVersionIndexes` `nil` to?
```suggestion
	knowVersions       []string
	knowVersionIndexes map[string]int
```

Reply:
this change is just by my IDE suggestion
```
Empty slice declaration via literal 
 Inspection info: Reports slice declaration with empty literal initializer instead of nil.
An empty slice can be represented by nil or an empty slice literal. The first approach is preferred as it does not lead to memory allocation.
See Code review comments: declaring empty slices.
```

`knowVersionIndexes  map[string]int` is not work

Reply:
this change is just by my IDE suggestion
```
Empty slice declaration via literal 
 Inspection info: Reports slice declaration with empty literal initializer instead of nil.
An empty slice can be represented by nil or an empty slice literal. The first approach is preferred as it does not lead to memory allocation.
See Code review comments: declaring empty slices.
```

`knowVersionIndexes  map[string]int` is not work